crates/hir_ty/src/lib.rs

```diff
@@ -566,7 +566,7 @@ mod tests {
 
                     stmt_str
                 }
-                hir::Stmt::ExprStmt {
+                hir::Stmt::Expr {
```

Comment:
**Code Structure Issue**: The renaming of 'hir::Stmt::ExprStmt' to 'hir::Stmt::Expr' might lead to confusion as 'ExprStmt' explicitly indicates an expression statement, whereas 'Expr' could be mistaken for an expression type. <br> **Fix**: Consider reverting the change or providing a more descriptive name that clearly differentiates between expressions and expression statements. <br> **Code Suggestion**: 
 ```
 -                hir::Stmt::ExprStmt {
 +                hir::Stmt::ExprStmt {
 ```